### PR TITLE
Add TA-88993

### PR DESCRIPTION
**Pull request by mdlinville**
Relates to [DOC-4576]

## Previews
[advisories/a88993.md](https://deploy-preview-15384--cockroachdb-docs.netlify.app/docs/advisories/a88993.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

**Review by rytaft — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15384#-:-NEbYFrNEqBsoZtXBJ1y:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @rmloveland)

File: advisories/a88993.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+The fix has been applied to the maintenance release of CockroachDB [v22.1.9](../releases/v22.1.html#v22-1-9).
+
+This public issue is tracked by [#88993](https://github.com/cockroachdb/cockroach/pull/88993).
+
+## Mitigation
+
+Users of CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.8](../releases/v22.1.html#v22-1-8) are encouraged to upgrade to [v22.1.9](../releases/v22.1.html#v22-1-9) or a later version. To determine whether your queries may be affected by this issue on v22.1.0 to v22.1.8, examine the query plans of any queries with an `ORDER BY` and `LIMIT` clause or aggregation with min or max by using an `EXPLAIN (OPT)` statement. In particular, you should examine plans for queries that have a multi-column index containing the `ORDER BY`, `min`, or `max` columns and for which a prefix of the index columns are constrained to two or more constant values by one of the following:
+
+- A query filter (e.g., WHERE a IN (1, 3))
```
Comment (rmloveland) [inline advisories/a88993.md:35]: suggest adding code font around the SQL e.g. `WHERE a in ...`

File: advisories/a88993.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- A `CHECK` constraint (e.g., CHECK (a IN (1, 3)))
```
Comment (rmloveland) [inline advisories/a88993.md:36]: same as above re: putting SQL in code font e.g. `CHECK (a IN ...)`

File: advisories/a88993.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- A computed column expression (e.g., `b INT AS (a + 10) STORED` given column a has filter `WHERE a IN (1, 3)`)
+- A `PARTITION` BY clause (e.g., `INDEX (a, ...) PARTITION BY LIST (a) (PARTITION p VALUES ((1), (3)))`)
```
Comment (rmloveland) [inline advisories/a88993.md:38]: need code font around all of `PARTITION BY`

File: advisories/a88993.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+If the plan shown by `EXPLAIN (OPT)` uses the multi-column index and shows a union-all, these queries may produce incorrect results.
```
Comment (rmloveland) [inline advisories/a88993.md:40]: suggest link from `EXPLAIN (OPT)` to https://www.cockroachlabs.com/docs/v22.1/explain#opt-option to save folks a search to do this check

**Pull request merged**